dot/rpc/modules/chain_test.go

```diff
@@ -15,3 +15,473 @@
 // along with the gossamer library. If not, see <http://www.gnu.org/licenses/>.
 
 package modules
+
+import (
+	"errors"
+	"math/big"
+	"net/http"
+	"testing"
+
+	apimocks "github.com/ChainSafe/gossamer/dot/rpc/modules/mocks"
+	"github.com/ChainSafe/gossamer/dot/types"
+	"github.com/ChainSafe/gossamer/lib/common"
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+)
+
+func TestChainModule_GetBlock(t *testing.T) {
```

Comment:
Can you run this (and others) with `t.Parallel()`? Let me know if they conflict with each other :wink: